### PR TITLE
fix(Dialog): text size

### PR DESCRIPTION
**Pull request by yceballost**
wrong text-presets was applied to title and description

### CODE REVIEW

File: src/dialog.tsx [hunk 1]
```diff
@@ -162,9 +162,9 @@ const Dialog: React.FC<DialogProps> = (props) => {
             {icon && <Box paddingBottom={24}>{icon}</Box>}
             {title && (
                 <Box paddingBottom={16}>
-                    <Text4 as="h2" light>
+                    <Text5 as="h2" light>
```
Comment (atabel) [inline src/dialog.tsx:165]: remove the light prop, Text5 can only be light
```suggestion
                    <Text5 as="h2">
```

**Review by atabel — COMMENTED** [review]
how did you manage to commit this with prettier errors? did you skip pre-commit hooks with `--no-verify`?
oh, maybe you commited from github?

**Comment by yceballost** [comment]
> how did you manage to commit this with prettier errors? did you skip pre-commit hooks with `--no-verify`? oh, maybe you commited from github?

yep! i commited your proposal directly from Github (is it something to avoid?)

**Review by atabel — APPROVED** [review]

**Comment by pladaria** [comment]
> yep! i commited your proposal directly from Github (is it something to avoid?)

yes, don't edit code using github

**Review by pladaria — APPROVED** [review]

**Pull request merged**